package.json

```diff
@@ -29,17 +29,79 @@
 		"Other"
 	],
 	"activationEvents": [
-		"onCommand:ocm-vscode-extension.ocmNewProject"
+		"onCommand:ocm-vscode-extension.ocmNewProject",
+		"onCommand:ocm-vscode-extension.verifyTools",
+		"onCommand:ocm-vscode-extension.getClusteradmVersion",
+		"onCommand:ocm-vscode-extension.buildLocalClusters",
+		"onView:ocm.explorer"
 	],
 	"main": "./out/src/extension.js",
 	"contributes": {
 		"commands": [
 			{
 				"command": "ocm-vscode-extension.ocmNewProject",
-				"title": "create an OCM application-manager project",
+				"title": "Create an application-manager project",
 				"category": "OCM"
+			},
+			{
+				"command": "ocm-vscode-extension.verifyTools",
+				"title": "Verify required tools existence",
+				"category": "OCM"
+			},
+			{
+				"command": "ocm-vscode-extension.getClusteradmVersion",
+				"title": "Get clusteradm version",
+				"category": "OCM"
+			},
+			{
+				"command": "ocm-vscode-extension.buildLocalClusters",
+				"title": "Build local clusters",
+				"category": "OCM"
+			}
+		],
+		"viewsContainers": {
+			"activitybar": [
+				{
+					"id": "ocmView",
+					"title": "OCM",
+					"icon": "images/icon-black.svg"
+				}
+			]
+		},
+		"views": {
+			"ocmView": [
```

Comment:
only the _forDevelopers_ view item has content.
maybe we should hide the other 3 until we implement them?